app/assets/javascripts/domain-config.js

```diff
@@ -0,0 +1,41 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.vars = window.GOVUK.vars || {}
+window.GOVUK.vars.extraDomains = [
+  {
+    name: 'production',
+    domains: [
+      'collections-publisher.publishing.service.gov.uk'
+    ],
+    initialiseGA4: true,
+    id: 'GTM-MG7HG5W',
+    gaProperty: 'UA-26179049-1',
+    gaPropertyCrossDomain: 'UA-145652997-1',
+    consentApiUrl: 'production'
```

Comment:
I don't think you'll need to define `consentApiUrl` for collections-publisher, as this relates to the single cookie consent code and it won't be using that. I think these lines can be safely removed.

Reply:
Thanks! Will remove these lines :)